src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -714,6 +718,14 @@ def exec_line(
                 validate_inputs=validate_inputs,
                 allow_generator_output=allow_generator_output,
             )
+            if line_result.run_info.status == Status.Completed:
```

Comment:
There will be cancelled case after we enable async flow test, please also handle the case.

Reply:
the status of span can only be okay/failed/notset. so I think for cases other than completed and failed, we may leave them as default (notset).